build/buildall

```diff
@@ -36,16 +36,19 @@ SPARK_SHIM_VERSIONS="
   313
   320
 "
-for sparkver in ${SPARK_SHIM_VERSIONS} ; do
+BUILD_PARALLELISM=1
```

Comment:
Should this accept an env var override?  e.g.:
```suggestion
BUILD_PARALLELISM=${BUILD_PARALLELISM-1}
```

Reply:
at this time we can't allow overriding parallelism. I started the work to make it possible https://github.com/NVIDIA/spark-rapids/pull/3523 showing significant boost. However, there are still issues in the CI left to be resolved.